ramls/schemas/fileDefinition.json

```diff
@@ -36,6 +36,16 @@
       "description": "Size of the file in Kbyte",
       "type": "integer"
     },
+    "format": {
+      "description": "Format of exported file",
```

Comment:
What do you mean by format of exported file, the output? If so why would csv be the default? our output default is mrc

Reply:
Ok, it makes sense, I will change it

Reply:
I still don't get it Illia, is it the format of the uploaded file or the file being exported?

Reply:
Format of an uploaded file, better to name uploadedFormat or something like this

Reply:
Guys, I don't think it matters, as for me it's the format of the file content, it can be used as we want, but to be consistent we should set it for all file definitions, and export file definition too. Therefore I added "mrc" type as well

Reply:
Please change the description, as this is filedefinition, please remove mrc to avoid any confusion, we will never allow for uploading mrc files